spec/mailers/previews/claim_mailer_preview.rb

```diff
@@ -31,10 +31,6 @@ def update_after_three_weeks_lup
     ClaimMailer.update_after_three_weeks(claim_for(Claim.approved.by_policy(Policies::LevellingUpPremiumPayments)))
   end
 
-  def email_verification
-    ClaimMailer.email_verification(claim_for(Claim.unsubmitted), OneTimePassword::Generator.new.code)
-  end
```

Comment:
do we use these mailer previews? none of them work for me locally.

Reply:
i don't use em, happy for them to go